hieradata_aws/class/frontend.yaml

```diff
@@ -1,8 +1,3 @@
 ---
 govuk::apps::collections::memcache_servers: 'frontend-memcached:11211'
-
-govuk::apps::contacts::db_hostname: 'mysql-replica'
-govuk::apps::contacts::db_username: 'contacts_fe'
-govuk::apps::contacts::db_password: "%{hiera('govuk::apps::contacts::db::mysql_contacts_frontend')}"
```

Comment:
It looks like these were originally added in 9e6193ec3c, at which point there was a `contacts` app on the frontend machines. 

Then in e6f575ab0 the `contacts` application was removed from the frontend machines, but these settings were left behind.

And now here they still are, doing nothing but confuse us.

Reply:
Cool, thanks - was gonna ask about that :D